UnityProject/Assets/Scripts/Systems/Research/ImporterExporter/TechwebJSONExporter.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Systems.Research.Data;
+
+namespace Systems.Research.ImporterExporter
+{
+	public class TechwebJSONExporter : TechwebExporter
+	{
+		public override Techweb Export(string filePath)
+		{
+			throw new System.NotImplementedException();
```

Comment:
the export would simply create a file and populate it with the object serialized to json, no?

Reply:
it would but I honestly have no idea what are the plans for the tool that will be used to deal managing and exporting this data and I've suggested multiple times now to not bloat the engine more than it already is and make this tool outside of Unity.